extensions/_beacons/http_status.py

```diff
@@ -101,37 +99,37 @@ def beacon(config):
         sites_config = _config['sites'][sites]
         url = sites_config['url']
         try:
-            if 'timeout' in sites_config:
-                r = requests.get(url, timeout=sites_config['timeout'])
-            else:
-                r = requests.get(url, timeout=30)
-        # r.status_code
+            r = requests.get(url, timeout=sites_config.get('timeout', 30))
         except requests.exceptions.RequestException as e:
-            log.debug("Request failed: %s", e)
-        if r.status_code >= 500:
-            log.debug('Response from status endpoint was invalid: '
-                      '%s', r.status_code)
+            log.info("Request failed: %s", e)
+            if r.raise_for_status:
+                log.info('Response from status endpoint was invalid: '
+                         '%s', r.status_code)
+                _failed = {'status_code': r.status_code}
```

Comment:
Don't forget to include the url in the event so that we can report on what failed :)